src/validations/constants.ts

```diff
@@ -143,6 +143,8 @@ export const OPTIONS = [
   'UNIT',
   'VARIABLE',
   'ICON',
+  'PARTIAL_REFERENCE',
```

Comment:
I don't think this is an option type? 🤔

Reply:
We have removed the PARTIAL_REFERENCE and the PARTIAL_INPUT_OBJECTS options.